packages/react-components/src/components/VideoTile.tsx

```diff
@@ -213,23 +222,56 @@ const videoTileMoreMenuProps = {
 };
 const VideoTileMoreOptionsButton = (props: {
   contextualMenu?: IContextualMenuProps;
+  participantDisplayName: string | undefined;
+  participantState: string | undefined;
+  participantHandRaised: boolean;
+  participantIsSpeaking: boolean | undefined;
+  participantIsMuted: boolean | undefined;
   canShowContextMenuButton: boolean;
 }): JSX.Element => {
-  const locale = useLocale();
+  const locale = useLocale().strings.videoTile;
   const theme = useTheme();
-  const strings = { ...locale.strings.videoTile };
 
-  const { contextualMenu, canShowContextMenuButton } = props;
+  const {
+    contextualMenu,
+    canShowContextMenuButton,
+    participantDisplayName,
+    participantHandRaised,
+    participantIsSpeaking,
+    participantState,
+    participantIsMuted
+  } = props;
+  const [moreButtonAiraDescription, setMoreButtonAriaDescription] = useState<string>('');
+
+  useEffect(() => {
+    setMoreButtonAriaDescription(
+      formatMoreButtonAiraDescription(
+        participantDisplayName,
+        participantIsMuted,
+        participantHandRaised,
+        participantState,
+        participantIsSpeaking,
+        locale
+      )
+    );
+  }, [
+    participantDisplayName,
+    participantHandRaised,
+    participantIsMuted,
+    participantIsSpeaking,
+    participantState,
+    locale
+  ]);
+
   if (!contextualMenu) {
     return <></>;
   }
 
   const optionsIcon = canShowContextMenuButton ? 'VideoTileMoreOptions' : undefined;
-
   return (
     <IconButton
       data-ui-id="video-tile-more-options-button"
-      ariaLabel={strings?.moreOptionsButtonAriaLabel}
+      ariaLabel={locale.moreOptionsButtonAriaLabel + ` ${moreButtonAiraDescription}`}
```

Comment:
```suggestion
      ariaLabel={locale.moreOptionsButtonAriaLabel + ` ${moreButtonAriaDescription}`}
```

Reply:
nit, strong concat can be dangerous with locale stuff. So make sure this makes coherent sense!